Mathlib/Topology/ContinuousOn.lean

```diff
@@ -636,6 +636,11 @@ theorem continuousWithinAt_congr_nhds {f : α → β} {s t : Set α} {x : α} (h
     ContinuousWithinAt f s x ↔ ContinuousWithinAt f t x := by
   simp only [ContinuousWithinAt, h]
 
+theorem ContinuousWithinAt.mono_nhdsWithin {f : α → β} {s t : Set α} {x : α}
```

Comment:
Currently, topology/derivatives mostly use [ContinuousWithinAt.mono_of_mem](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/ContinuousOn.html#ContinuousWithinAt.mono_of_mem) spelling.

Reply:
I changed the proof to use `ContinuousWithinAt.mono_of_mem` instead, and deleted this new lemma.